src/main/java/name/webdizz/sonar/grammar/utils/SpellCheckerUtil.java

```diff
@@ -0,0 +1,193 @@
+package name.webdizz.sonar.grammar.utils;
+
+import com.swabunga.spell.engine.Configuration;
+import com.swabunga.spell.engine.SpellDictionaryHashMap;
+import com.swabunga.spell.engine.Word;
+import com.swabunga.spell.event.SpellChecker;
+import com.swabunga.spell.event.StringWordTokenizer;
+import name.webdizz.sonar.grammar.exceptions.UnableToLoadDictionary;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Provides utility methods for spell checking. Uses an open source API
+ * <a href="http://jazzy.sourceforge.net/">Jazzy</a> for spell checking.
+ */
+public class SpellCheckerUtil{
+  private static SpellDictionaryHashMap dictionary = null;
+  private static SpellChecker spellChecker = null;
+
+  static{
```

Comment:
not a good solution, please extract dictionary loading into simple class dedicated for loading, and make util less dependant on static dictionary loading